package.json

```diff
@@ -27,5 +30,9 @@
     "postcss": "^8",
     "tailwindcss": "^3.3.0",
     "typescript": "^5"
-  }
+  },
```

Comment:
- Remove the author, license, repository and main keys
- Kindly rename the images with proper name.